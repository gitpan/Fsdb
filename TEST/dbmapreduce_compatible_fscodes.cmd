prog='dbmapreduce'
args='-k experiment dbcolstats -F S duration'
cmp='diff -cb '
subprogs=dbcolstats
in=TEST/dbmapreduce_incompatible_fscodes.in
suppress_warnings='5.1[0-2]:Unbalanced string table refcount;5.1[0-2]:Scalars leaked'
