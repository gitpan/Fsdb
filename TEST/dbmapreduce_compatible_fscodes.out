#fsdb -F S experiment mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n
ufs mab sys  37.25  0.070711  0.18983  0.6353  36.615  37.885  0.95  74.5  2775.1  37.2  37.3  2
ufs rcp real  271.2  9.4752  3.4938  85.13  186.07  356.33  0.95  542.4  1.4719e+05  264.5  277.9  2
# test trailing comment
#   | dbmapreduce -k experiment -- dbcolstats --nolog -F S duration
